Code/ForceField/ANI/AtomicContrib.cpp

```diff
@@ -132,7 +133,39 @@ double ANIAtomContrib::getEnergy(Eigen::ArrayXXd &aev) const {
   return this->ANIAtomContrib::forwardProp(row) + this->d_selfEnergy;
 }
 
-void ANIAtomContrib::getGrad(double *pos, double *grad) const {}
+void ANIAtomContrib::getGrad(double *pos, double *grad) const {
+  auto initEnergy = this->dp_forceField->calcEnergy(pos);
+
+  // + - x movement
+  pos[3 * this->d_atomIdx] += 1e-5;
```

Comment:
Please make the displacement amount a const so that you don't have to keep repeating it.

Reply:
How did you pick the value `1e-5`?